src/main/java/com/exasol/adapter/document/edml/deserializer/DeserializationHelper.java

```diff
@@ -56,4 +56,7 @@ static Optional<Boolean> readOptionalBoolean(final JsonObject json, final String
             return Optional.empty();
         }
     }
+    public static String jsonObjectToString(JsonObject jsonObject){
+return jsonObject.toString();
```

Comment:
Strange indent .... 
Maybe formatter is not activated?

Reply:
good point, didn't run the formatter on  this file